### PR TITLE
Devop/include one inch

### PR DESCRIPTION
**Pull request by SteveMieskoski**
### Devop

* \[x] Updated CHANGELOG.md
* \[x] Add PR label
* \[x] No hard-coded strings

### CODE REVIEW

**Review by gamalielhere — APPROVED** [review]

**Comment by gesseekur** [comment]
@SteveMieskoski can you fix build so i can test hehe

File: src/partners/oneInch/oneInch-calls.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+};
+
+const createTransaction = async transactionParams => {
+  try {
+    const results = await get(
+      `${buildPath(GET_TRADE)}?address=${
+        transactionParams.fromAddress
+      }&recipient=${
+        transactionParams.toAddress
+      }&dex=ONE_INCH&exchange=one_inch&platform=ios&fromContractAddress=${
```
Comment (kvhnuke) [inline src/partners/oneInch/oneInch-calls.js:87]: `platform=web`

**Review by kvhnuke — CHANGES_REQUESTED** [review]

File: compileBaseSwapOptions.js [hunk 2]
```diff
@@ -110,6 +110,8 @@ class CompileSwapOptions {
       );
       const tokenDetails = {};
       for (let i = 0; i < tickers.tickers.length; i++) {
+        // waits for half a sec to call coingecko to avoid rate limit
+        setTimeout(function () {}, 500);
```
Comment (kvhnuke) [inline compileBaseSwapOptions.js:114]: this does not wait half a sec, `await new Promise(r => setTimeout(r, 500));`

File: src/layouts/InterfaceLayout/containers/SwapContainer/components/ProvidersRadioSelector/ProviderInfoList/ProviderInfoList.vue [hunk 3]
```diff
@@ -86,6 +88,7 @@ export default {
       );
     },
     getTagLine(name) {
+      console.log(name); // todo remove dev item
```
Comment (kvhnuke) [inline src/layouts/InterfaceLayout/containers/SwapContainer/components/ProvidersRadioSelector/ProviderInfoList/ProviderInfoList.vue:91]: remove console logs

**Review by kvhnuke — CHANGES_REQUESTED** [review]

**Review by kvhnuke — APPROVED** [review]

**Pull request merged**